Starbucks/Controllers/ProductViewController/Components/ProductSizeTableCell.swift

```diff
@@ -48,15 +57,38 @@ class ProductSizeTableCell: UITableViewCell {
     
     func setupUI() {
         addSubview(sizePriceStackView)
+        addSubview(checkMarkButton)
         
         sizePriceStackView.leftAnchor.constraint(equalTo: leftAnchor, constant: 20).isActive = true
         sizePriceStackView.centerYAnchor.constraint(equalTo: centerYAnchor).isActive = true
-
         
+        checkMarkButton.rightAnchor.constraint(equalTo: rightAnchor, constant: -20).isActive = true
+        checkMarkButton.centerYAnchor.constraint(equalTo: centerYAnchor).isActive = true
     }
-
     
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+    
+    @objc func buttonAction() {
+        var currentImage = checkMarkButton.currentImage
+        if currentImage == checkMarkButton.imageView?.image {
+             currentImage = UIImage(named: "unSelectedCheckMark")
+                checkMarkButton.setImage(UIImage(named: "selectedCheckmark"), for: .normal)
+        } else {
+                checkMarkButton.setImage(UIImage(named: "unSelectedCheckMark"), for: .normal)
+            }
+        }
 }
```

Comment:
I'll show you how to do this part. It will be controlled from the ViewController with the `didSelectItemAtIndexPath` function

Reply:
so must I delete it or leave it like that